nixos/modules/testing/test-instrumentation.nix

```diff
@@ -109,6 +109,12 @@ in
       # Allow very slow start
       DefaultTimeoutStartSec=300
     '';
+    systemd.user.extraConfig = ''
+      # Don't clobber the console with duplicate systemd messages.
+      ShowStatus=no
```

Comment:
Won’t this make it harder to see the process status from logs?

Reply:
No idea, I just copied the config above for the system manager. Would things be different for the user manager?

Reply:
Sorry, did not notice it in the unexpanded diff. I would half expect the user values to be inherited :woman_shrugging:

Not sure if it should be there for system in the first place. The introducing commit https://github.com/ncfavier/nixpkgs/commit/eab25b104ad12b75a8b9a41ce3ac8b3b931144ef does not have much context.

Reply:
They're not inherited, I checked in a VM and the TimeoutStart was 90 seconds, hence the failure on aarch64.

Also it seems like ShowStatus actually only applies to the system manager.